pages/Useful Utilities/Hyprland-desktop-portal.md

```diff
@@ -38,29 +38,28 @@ yay -S xdg-desktop-portal-hyprland-git
 ## Unmask dependencies
 ### /etc/portage/profile/package.unmask
 ```plain
-=dev-qt/qtbase-6.4.0
-=dev-qt/qtwayland-6.4.0
-=dev-qt/qtdeclarative-6.4.0
-=dev-qt/qtshadertools-6.4.0
+=dev-qt/qtbase-6.4.3
+=dev-qt/qtwayland-6.4.3
+=dev-qt/qtdeclarative-6.4.3
+=dev-qt/qtshadertools-6.4.3
```

Comment:
Matching on version is not needed in this case. So:
```dev-qt/qtbase
dev-qt/qtwayland
dev-qt/qtdeclarative
dev-qt/qtshadertools
```

Reply:
Done both. Is it good now?